xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-docker/src/test/it/org/xwiki/ckeditor/test/ui/ImagePluginIT.java

```diff
@@ -549,17 +573,82 @@ void updateExternalImageSize(TestUtils setup, TestReference testReference) throw
 
         assertEquals("[[image:" + imageURL + "||height=\"50\" width=\"50\"]]", savedPage.editWiki().getContent());
     }
+    
+    @Test
+    @Order(13)
+    void quickInsertImageThisPage(TestUtils setup,
+            TestReference testReference) throws Exception
+    {
+        
+        // Upload an attachment to the page.
+        String attachmentName = "image.gif";
+        AttachmentReference attachmentReference = new AttachmentReference(attachmentName, testReference);
+        uploadAttachment(setup, testReference, attachmentName);
+        
+        // Move to the WYSIWYG edition page.
+        edit(setup, testReference, false);
+        
+        // Run the image quick action.
+        textArea.sendKeys("/image");
+        AutocompleteDropdown qa = new AutocompleteDropdown().waitForItemSelected("/image", "Image");
+        textArea.sendKeys(Keys.ENTER);
+        qa.waitForItemSubmitted();
+        
+        // Select the newly uploaded image.
+        textArea.sendKeys("image");
+        AutocompleteDropdown img = new AutocompleteDropdown().waitForItemSelected("img::image", attachmentName);
+        textArea.sendKeys(Keys.ENTER);
+        img.waitForItemSubmitted();
+        
+        assertSourceEquals("[[image:" + setup.serializeReference(attachmentReference) + "]]");
```

Comment:
Using absolute reference until we fix [XWIKI-21182](https://jira.xwiki.org/browse/XWIKI-21182).